internal/plugin/plugin.go

```diff
@@ -77,7 +78,11 @@ func GetAllValidPlugins(existingCommands []*cobra.Command) []*Plugin {
 	// Convert manifests to plugins
 	plugins := make([]*Plugin, 0, len(manifests))
 	for _, manifest := range manifests {
-		plugins = append(plugins, createPluginFromManifest(manifest))
+		plugin, err := createPluginFromManifest(manifest)
+		if err != nil {
+			logPluginWarning(err.Error())
```

Comment:
shouldn't this be:
```go
if err != nil {
    logPluginWarning(err.Error())
    continue
}
```

you don't want to add a plugin to the list if `err != nil`

Reply:
yeah, oversight ....